src/Microsoft.DocAsCode.Build.Engine/Incrementals/IncrementalBuildContext.cs

```diff
@@ -363,6 +366,21 @@ public void InitFileAttributes()
             }
         }
 
+        public void LoadFileMetadataChanges()
+        {
+            var changedGlobs = CurrentBuildVersionInfo.FileMetadata.GetChangedGlobs(LastBuildVersionInfo.FileMetadata).ToArray();
+            foreach (var file in LastBuildVersionInfo.Attributes.Keys)
+            {
+                if (changedGlobs.Any(g => g.Match(file)))
+                {
+                    if (_changeDict[file] == ChangeKindWithDependency.None)
+                    {
+                        _changeDict[file] = ChangeKindWithDependency.Updated;
```

Comment:
>Updated [](start = 69, length = 7)

would it break things if the file is actually deleted in the change list?

Reply:
If a file is deleted, the value is `Deleted`, and I will keep it `Deleted` here.